### PR TITLE
publishing to edge https://sitecore.atlassian.net/browse/ACC-12

### PR DESCRIPTION
**Pull request by mixxr**
## Description / Motivation

Added Publishing to Edge Recipe

[https://sitecore.atlassian.net/browse/ACC-12](https://sitecore.atlassian.net/browse/ACC-12)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update (non-breaking change; modified files are limited to the `/data` directory or other markdown files)

## Checklist:

- [ ] I have read the Contributing guide.
- [ ] My code/comments/docs fully adhere to the Code of Conduct.
- [ ] My change is a code change.
- [ ] My change is a documentation change and there are NO other updates required.
- [ ] My change has new or updated images which are stored in the `/public/images` folder that need to be migrated to Sitecore DAM

### CODE REVIEW

**Comment by robearlam** [comment]
@mixxr  - there are merge conflicts on this PR, can they be fixed so we can review?

File: data/markdown/pages/learn/accelerate/xm-cloud/pre-development/information-architecture/publishing-to-edge.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    name
+    mFields: fields {
+      name
+      jsonValue
+    }
+  }
+}
+```
+
+It is required to already have published - “/sitecore/content/<coll>/<site>/Home/About/Data/Text 1” . The connector is not aware about the relationships, so it cannot publish integrated GraphQL related items at publishing time.
```
Comment (robearlam) [inline data/markdown/pages/learn/accelerate/xm-cloud/pre-development/information-architecture/publishing-to-edge.md:150]: These HTML tags need to be escaped as they are breaking the build

**Review by robearlam — CHANGES_REQUESTED** [review]
@mixxr HTML Tags in one of the MD files aren't escaped and are breaking the build.

These need to be fixed before it can be merged.

File: data/markdown/pages/learn/accelerate/xm-cloud/pre-development/information-architecture/publishing-to-edge.md (hunk 1 above)
Comment (mixxr) [reply data/markdown/pages/learn/accelerate/xm-cloud/pre-development/information-architecture/publishing-to-edge.md]: fixed

**Review by robearlam — APPROVED** [review]

**Pull request merged**